### DISCUSSION TITLE
Pyright wont start

### DISCUSSION THREAD

**Discussion by PCesteban** [LunarVim/LunarVim#2827] [Q&A] [answered]
Hello, I just installed Lunarvim and wanted to edit a python file. But pyright after it was installed automatically, don't start and I received this message: 
![image](https://user-images.githubusercontent.com/48070832/179574894-2a322032-7b93-4180-b18e-a9d1655c8542.png)

I'm unable to use pyright inside Lunarvim, could someone help me with this issue? I really appreciate any help you can provide.

![image](https://user-images.githubusercontent.com/48070832/179575546-9d5caec2-154b-4aca-9aca-f3464b61e43f.png)

**Comment by rebuilt** [comment, marked as answer]
The LSP error logs might provide more clues.  I'm on my phone so I don't remember the shortcut to view the logs.  You can find it in the whichkey menu.  The shortcut is something like Space + Ll

**Reply by PCesteban** [reply]
I made it work by updating npm and nodeJS, I think that was a requirement, but now I don't know how to make Black work as the default formatter for python files, I uncommented the section that is supposed to enable this in the config file, but it does not work with the LSP.

![image](https://user-images.githubusercontent.com/48070832/180262248-ad62b117-4c5d-4d84-9d05-8432b515d703.png)

I already have Black installed.

**Reply by rebuilt** [reply]
Can you switch it to the short version to see if it makes a difference?


```lua
  local formatters = require("lvim.lsp.null-ls.formatters")
  formatters.setup({
    { exe = "black" },
  })

```

**Reply by PCesteban** [reply]
yes, now is working, but I had to uninstall and reinstall black. Thank you!!
